### PR TITLE
Change the positon of donation banner

### PR DESCRIPTION
**Pull request by therealharshit**
@pikurasa since the current position of donation banner is causing navbar to break, I have changed the position of donation banner.

This is how it looks now:
Screenshot 2024-12-12 022042
I'll hope it's an improvement.

### CODE REVIEW

**Comment by pikurasa** [comment]
It's a start. However:

1. It blocks some of the text at the top of the page; and
2. It has a bit of a gap upon scrolling down the page

Below are screenshots:
Screenshot from 2024-12-13 13-22-51
Screenshot from 2024-12-13 13-22-46

**Comment by therealharshit** [comment]
> It's a start. However:
> 
> 1. It blocks some of the text at the top of the page; and
> 2. It has a bit of a gap upon scrolling down the page
> 
> Below are screenshots:
> Screenshot from 2024-12-13 13-22-51
> Screenshot from 2024-12-13 13-22-46
> 

@pikurasa Thanks for reviewing.
 I can also resolve that in this PR but:
1. Is resolved in #577
2. Is resolved in #578

**Comment by therealharshit** [comment]
> It's a start. However:
> 
> 1. It blocks some of the text at the top of the page; and
> 2. It has a bit of a gap upon scrolling down the page
> 
> Below are screenshots:
> Screenshot from 2024-12-13 13-22-51
> Screenshot from 2024-12-13 13-22-46
> 

@pikurasa thanks for reviewing, I can resolve those issue but there are already PR raised which resolves those issue:
1. Is resolve in #578
2. Will be resolved after merging #587

Thanks.

**Comment by pikurasa** [comment]
@therealharshit This branch now has conflicts that will need to be resolved.

Also, I'm awaiting a response for requested updates to https://github.com/sugarlabs/www/pull/587

File: css/custom.css [hunk 1]
```diff
@@ -15,3 +15,9 @@
 .intro_img>.container>div>div>div>p {
   padding: 0 10px;
 }
+
+.donation-bar {
+  background-color: white;
+  margin-top: 90px;
+  padding: 10px 0; 
+}
```
Comment (quozl) [inline css/custom.css:23]: Missing newline at end of file.

**Comment by Devmoni** [comment]
@therealharshit I appreciate your efforts on this, but I feel this method might not work well on mobile screens since the donation banner takes up almost half of the space. I had also tried similar changes to the files earlier but wasn’t fully satisfied with the outcome.

**Comment by Devmoni** [comment]
@pikurasa @quozl 
Should we consider fixing the position of the banner on the screen, so it’s always visible, or just display it at the top?

I’m a bit concerned that keeping it fixed might create issues on smaller screens or mobile devices. Could you please guide me on the best approach for this?

**Comment by therealharshit** [comment]
> @therealharshit This branch now has conflicts that will need to be resolved.
> 
> Also, I'm awaiting a response for requested updates to #587

@pikurasa I have done a rebase to resolve the conflicts.
@quozl please review.

**Comment by therealharshit** [comment]
> @therealharshit I appreciate your efforts on this, but I feel this method might not work well on mobile screens since the donation banner takes up almost half of the space. I had also tried similar changes to the files earlier but wasn’t fully satisfied with the outcome.

@Devmoni Thanks for the review, but this PR only change the position of donation banner for desktop screen sizes as it was breaking the navbar on desktop.

**Comment by therealharshit** [comment]
> @pikurasa @quozl Should we consider fixing the position of the banner on the screen, so it’s always visible, or just display it at the top?
> 
> I’m a bit concerned that keeping it fixed might create issues on smaller screens or mobile devices. Could you please guide me on the best approach for this?

For smaller screens, I think we should add an option to minimize the banner, so that people can minimize it once they have gone through it and then continue to site.
@pikurasa what do you think?

**Comment by pikurasa** [comment]
> For smaller screens, I think we should add an option to minimize the banner, so that people can minimize it once they have gone through it and then continue to site.
> @pikurasa what do you think?

Yes, that sounds reasonable.

**Pull request merged**